### DISCUSSION TITLE
Proposta de PyLadies Conf online

### DISCUSSION THREAD

**Discussion by anicelysantos** [pyladies-brazil/organizacao#91] [General]
Olá!

Estava pensando sobre a possibilidade de continuarmos fazendo a Conferência, agora de maneira online. Talvez essa seja uma maneira de garantir que exista anualmente, pois não precisaria depender do capítulo local e qualquer pessoa de qualquer lugar poderia participar e/ou ajudar na org. 

A ideia dessa discussão é saber se mais pessoas animam da gente puxar uma reunião e pensar em algumas ações nesse sentido.

Vamos gente, pessoas!

**Comment by emillyhorta** [comment]
Tô dentro!

**Comment by anadulce** [comment]
Acho a ideia genial <3 E também acho que faz falta um momento de interação entre PyLadies, da forma que ocorria no Papo entre PyLadies.

**Comment by lidymonteiro** [comment]
tem meu apoio! vamos fazer acontecer

**Reply by akrs3** [reply]
Super apoio a ideia!
